test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,267 @@
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... successfully ran');
+    if (successNotificationWasFound != true) {
+      const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Create a Default Scratch Org... failed to run');
+      if (failureNotificationWasFound == true) {
+        if (await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'organization has reached its daily scratch org signup limit', 5)) {
+          // This is a known issue...
+          utilities.log('Warning - creating the scratch org failed, but the failure was due to the daily signup limit');
+        } else if (await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'is enabled as a Dev Hub', 5)) {
+          // This is a known issue...
+          utilities.log('Warning - Make sure that the org is enabled as a Dev Hub.');
+          utilities.log('Warning - To enable it, open the org in your browser, navigate to the Dev Hub page in Setup, and click Enable.');
+          utilities.log('Warning - If you still see this error after enabling the Dev Hub feature, then re-authenticate to the org.');
+        } else {
+          // The failure notification is showing, but it's not due to maxing out the daily limit.  What to do...?
+          utilities.log('Warning - creating the scratch org failed... not sure why...');
+        }
+      } else {
+        utilities.log('Warning - creating the scratch org failed... neither the success notification or the failure notification was found.');
+      }
+    }
+    expect(successNotificationWasFound).toBe(true);
+
+    // Look for orgAliasName in the list of status bar items.
+    const statusBar = await workbench.getStatusBar();
+    const scratchOrgStatusBarItem = await utilities.getStatusBarItemWhichIncludes(statusBar, scratchOrgAliasName);
+    expect(scratchOrgStatusBarItem).not.toBeUndefined();
+  });
+
+  step('Run SFDX: Set a Default Org', async () => {
+    const workbench = await browser.getWorkbench();
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Set a Default Org', 1);
+
+    let scratchOrgQuickPickItemWasFound = false;
+    const currentOsUserName = await utilities.currentOsUserName();
+    const quickPicks = await inputBox.getQuickPicks();
+    for (const quickPick of quickPicks) {
+      const label = await quickPick.getLabel();
+      if (scratchOrgAliasName) {
+        // Find the org that was created in the "Run SFDX: Create a Default Scratch Org" step.
+        if (label.includes(scratchOrgAliasName)) {
+          await quickPick.select();
+          await utilities.pause(3);
+          scratchOrgQuickPickItemWasFound = true;
+          break;
+        }
+      } else {
+        // If the scratch org was already created (and not deleted),
```

Comment:
I'm a bit confused about this scenario - why would this one be necessary? Can discuss offline too

Reply:
Discussed this, Jeff to look into this one further

Reply:
@randi274 I looked into it and I'm not seeing this condition being used anymore, and removed it from the two locations.